scripts/reboot

```diff
@@ -78,6 +79,12 @@ function stop_sonic_services()
     stop_pmon_service
 }
 
+function stop_services_asan()
+{
+    debug "Stopping swss for ASAN"
+    systemctl stop swss
```

Comment:
Want to confirm here that SWSS needs to be stopped on all platforms, not just on Mellanox

Reply:
It's for all platforms since asan for swss is platform-independent.